apps/sw_web/src/sw_web_app.erl

```diff
@@ -44,14 +46,15 @@ start(_StartType, _StartArgs) ->
                        %% max limit have any effect since the socket might make the entire
                        %% HTTP request available when cowboy does a gen_tcp:read(Socket, 0)
                        %% and will ignore the limit.
-                       {max_request_line_length, 65536},
+                       %% {max_request_line_length, 65536},
```

Comment:
This change might hit us, but I don't think I'll worry too much about it for now.

The problem is that GraphiQL uses the URL to store the query document, so it gets pretty big quickly. It is nice when you can send a URL to someone else and they can execute the same query easily, so we've had lots of use of this feature.

Reply:
Ah yes - completely forgot about this. As far as I can see these options need can now be passed when changing the handler as 4th arg here: https://github.com/shopgun/graphql-erlang-tutorial/blob/master/apps/sw_web/src/sw_web_graphql_handler.erl#L29